src/server/scripts/Northrend/Naxxramas/boss_gluth.cpp

```diff
@@ -35,21 +38,28 @@ enum Creatures
 
 Position const PosSummon[3] =
 {
-    {3267.9f, -3172.1f, 297.42f, 0.94f},
-    {3253.2f, -3132.3f, 297.42f, 0},
-    {3308.3f, -3185.8f, 297.42f, 1.58f},
+    { 3267.9f, -3172.1f, 297.42f, 0.94f },
+    { 3253.2f, -3132.3f, 297.42f, 0 },
+    { 3308.3f, -3185.8f, 297.42f, 1.58f },
 };
 
 enum Events
 {
-    EVENT_WOUND     = 1,
+    EVENT_WOUND = 1,
     EVENT_ENRAGE,
     EVENT_DECIMATE,
     EVENT_BERSERK,
     EVENT_SUMMON,
+    EVENT_SEARCH_ZOMBIE,
+    EVENT_KILL_ZOMBIE,
 };
 
-#define EMOTE_NEARBY    " spots a nearby zombie to devour!"
+#define EMOTE_NEARBY    "Gluth spots a nearby zombie to devour!"
+#define EMOTE_DECIMATE  "Gluth decimates all nearby flesh!"
+#define EMOTE_ENRAGE    "Gluth becomes enraged!"
+
+#define SECONDS *1000
+#define MINUTES *60000
```

Comment:
To be honest,
```C++
events.ScheduleEvent(EVENT_SUMMON, 15 SECONDS);
events.ScheduleEvent(EVENT_BERSERK, 8 MINUTES);
```
 is far more clear to read and write than 
```C++
events.ScheduleEvent(EVENT_SUMMON, 15 * IN_MILLISECONDS);
events.ScheduleEvent(EVENT_BERSERK, 8  * MINUTE * IN_MILLISECONDS);
```
respectively.  It also add this abstraction where the scripter doesn't have to remember that the second parameter of ScheduleEvent() is in milliseconds. And stuff like this is also possible AND clean at the same time:

```C++
#define MILLISECONDS *1
#define SECONDS *1000
#define MINUTES *60000
#define HOURS *60*60000

events.ScheduleEvent(EVENT_BERSERK, 2 MINUTES + 25 SECONDS);
```

But I get that it may create name conflicts and also that making the change would mean changing every script which I'm not volunteering to do at the moment :p So, I'll make the change if needed.